src/elements/featured-videos.html

```diff
@@ -113,44 +113,48 @@
     </style>
 
     <div class="container">
-      <div class="videos-header">
-        <h4 class="heading">{$ featuredVideosTitle $}</h4>
-        <div class="controls">
-          <paper-icon-button icon="icons:chevron-left" on-tap="shiftContentLeft"></paper-icon-button>
-          <paper-icon-button icon="icons:chevron-right" on-tap="shiftContentRight"></paper-icon-button>
+      <template is="dom-if" if="[[_videos]]">
```

Comment:
There are several sources of truth here. One is firebase and the other is a configuration.
I think we should display the component if we have `videosSessionsUrl` defined in config regardless we have data in the firebase or not.